packages/s3-cache/src/index.ts

```diff
@@ -0,0 +1,62 @@
+import { S3 } from '@aws-sdk/client-s3'
+import { getAWSCredentials, type CacheEntry, type CacheStrategy } from '@dbbs/next-cache-handler-common'
+
+export class S3Cache implements CacheStrategy {
```

Comment:
Does it make sense to use extra-memory cache here? 
The one from memory strategy or 
https://github.com/DBB-Software/platform/blob/main/packages/common/src/domain/setting-service-client.ts
for mainly de-bouncing

Reply:
not sure that it needs to be there, it is more like hybrid strategy already, but i am thinking that we can make it as flag for s3 strategy to extra level of caching. But again. based on the project's size memory cache won't be really proactive at this point